cypress.config.ts

```diff
@@ -0,0 +1,43 @@
+import { defineConfig } from "cypress";
+
+export default defineConfig({
+  defaultCommandTimeout: 10000,
+  e2e: {
+    // We've imported your old cypress plugins here.
+    // You may want to clean this up later by importing these.
+    setupNodeEvents(on, config) {
+      on("task", {
+        log(args) {
+          console.log(args);
+
+          return null;
+        },
+        table(message) {
+          console.table(message);
+
+          return null;
+        },
+      });
+      return config;
+    },
+    baseUrl: "http://0.0.0.0:8400",
+    specPattern: "cypress/e2e/**/*.{js,jsx,ts,tsx}",
+    viewportHeight: 1300,
+    viewportWidth: 1440,
+  },
+  env: {
+    BASENAME: "/MAAS",
+    REACT_BASENAME: "/r",
```

Comment:
I've added the basename env variables here, whereas before they were inferred from the `.env` file

Reply:
Is this because v10 doesn't support env vars for this config?

Reply:
Kind of. We would need to prefix the vars in `.env` with `CYPRESS_` in order for them to get parsed by cypress, but at that point we might as well keep all the cypress-specific stuff in this one config file.